src/components/Taxonomy/TaxonomyLightbox.tsx

```diff
@@ -110,13 +113,13 @@ const TaxonomyLightbox = ({
           </StyledHeader>
           <StyledContent>
             {children}
-            {onSelect && (
-              <StyledWrapper>
-                <ButtonV2 onClick={onSelect} data-testid="taxonomyLightboxButton">
-                  {loading ? <Spinner appearance="small" /> : t('form.save')}
+            <StyledWrapper>
+              {actions.map((a, i) => (
+                <ButtonV2 key={i} {...pick(a, 'onClick', 'data-testid')}>
```

Comment:
Hva skjer her?

Reply:
Velger kun de attributtene som skal sendes til Button

Reply:
Ser den, men vi trenger ikke å bruke lodash for å få til det!

Reply:
Det er et veldig godt poeng😂